.github/container/Dockerfile.base

```diff
@@ -207,26 +160,14 @@ ADD nccl-sanity-check.cu /opt
 RUN install-nccl-sanity-check.sh
 ADD jax-nccl-test parallel-launch /usr/local/bin/
 
-###############################################################################
-## Add the systemcheck to the entrypoint.
-###############################################################################
-
-COPY check-shm.sh /opt/nvidia/entrypoint.d/
-
-###############################################################################
-## Add the GCP - TCPX check to the entrypoint.
-###############################################################################
-
-# TODO(chaserileyroberts): Reenable once fully tested on GCP.
-# COPY gcp-autoconfig.sh /opt/nvidia/entrypoint.d/
```

Comment:
Not sure about this one. Better ask Yu-Hang/Chase whether we need this or not.

Reply:
It is related to TCPX, but it got disabled almost immediately after it was added.
@yhtang to comment on the next steps given his recent work.

Reply:
It only applies to the A3 nodes and a different configuration is needed for A3 mega. Given that Google's interconnect technology is still quickly evolving, I now think we should refrain from shipping this type of autoconfig scripts until the API stabilizes. For the time being, we can add a docs page and link to GCP's own configuration guide.